nginx.conf

```diff
@@ -25,6 +25,436 @@ http {
     add_header "X-Frame-Options" "DENY" always;
     add_header "X-Content-Type-Options" "nosniff";
     add_header "X-Permitted-Cross-Domain-Policies" "master-only";
+
+    rewrite ^/documentation/*$ http://documentation-production.s3-website-us-east-1.amazonaws.com/documentation/* permenant;
```

Comment:
This should be `permanent` not `permenant`, right?

Reply:
Yes. There's also a problem with line 456 (I think we need to remove the `302!`). I'm working on those changes now but for some reason nginx still isn't responding to requests